<launch>
  <arg name="device_name"              default="/dev/ttyUSB0"/>
  <arg name="baud_rate"                default="1000000"/>

  <arg name="scan_range"               default="10"/>

  <arg name="pan_id"                   default="1"/>
  <arg name="tilt_id"                  default="2"/>

<!--ADD-->
  <arg name="position_p_gain"	       default="800"/>
  <arg name="position_i_gain" 	       default="0"/>
  <arg name="position_d_gain"	       default="0"/>

<!--end-->
  <param name="device_name"            value="$(arg device_name)"/>
  <param name="baud_rate"              value="$(arg baud_rate)"/>

  <param name="scan_range"             value="$(arg scan_range)"/>

  <param name="position_p_gain"	       value="$(arg position_p_gain)"/>
  <param name="position_i_gain"	       value="$(arg position_i_gain)"/>
  <param name="position_d_gain"	       value="$(arg position_d_gain)"/>

  <node name="torque_ctrl_6DOF" pkg="dynamixel_workbench_controllers" type="torque_ctrl_6DOF" required="true" output="screen"/>
</launch>
